package.json

```diff
@@ -6,17 +6,16 @@
   "dependencies": {
     "express": "^4.14.0"
   },
+  "peerDependencies" : {
+    "bs-platform": "file:../bucklescript.git/"
+  },
   "devDependencies": {
-    "bs-platform": "^1.0.1"
+    "bs-platform": "file:../bucklescript.git/"
```

Comment:
As I understand it, it should not. I believe `peerDependencies` is for subpackages that are exposed through this package. E.g. if package `foo` exposes the package `bar` via `foo.bar`, `bar` should be a `peerDependency` of `foo`. But my understanding of this admittedly isn't great.